packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest-mix/expected.js

```diff
@@ -0,0 +1,11 @@
+function fn(a1) {
+  var a2 = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : 4;
+  var _arguments$ = arguments[2],
+      a3 = _arguments$.a3,
+      a4 = _arguments$.a4;
+  var a5 = arguments[3];
+
+  var _ref = arguments.length > 4 && arguments[4] !== undefined ? arguments[4] : {},
```

Comment:
Is the `.length` check necessary? If the argument is not present it will be `undefined` anyway.